src/main/scala/tv/teads/github/api/http/OkHttpClientWrapper.scala

```diff
@@ -19,6 +20,7 @@ private[api] class OkHttpClientWrapper(authenticator: Option[Authenticator], max
   def executeAsync(requestBuilder: Request.Builder, customAuthenticator: Option[Authenticator]): Future[Response] = {
     val auth = customAuthenticator orElse authenticator
     val authenticatedRequest = auth.map(_(requestBuilder)).getOrElse(requestBuilder).build()
+    logger.debug(s"Calling Github : url ${authenticatedRequest.url()}")
```

Comment:
add the method too, only the URL is not sufficient.
Adding the body could be useful too